src/Http/Response.php

```diff
@@ -1197,6 +1197,62 @@ public function withAddedLink(string $url, array $options = [])
         return $this->withAddedHeader('Link', '<' . $url . '>' . $param);
     }
 
+    /**
+     * Get an array representation of the HTTP Link header values
+     *
+     * @return array
+     */
+    public function getParsedLink(): array
```

Comment:
Since it returns an array `getParsedLinks()` would be better. That said instead of adding a new method to `Response` I am wonder if instead we should create a separate helper class for parsing specific header types.

Reply:
For reference https://github.com/guzzle/guzzle/blob/master/docs/psr7.rst#complex-headers

Reply:
Are there other header parsing examples already present in Cake which would justify spliting this up in a separate helper?
There is the `WWW-Authenticate` Header which is used in the Digest Auth that comes to my mind.

Reply:
https://github.com/cakephp/cakephp/blob/4.x/src/Http/ContentTypeNegotiation.php

Reply:
Should we tackle that extraction in this PR or in a separate PR?

Reply:
I would rather we go a clean implementation from the get go.